qatrfm/cli.py

```diff
@@ -76,9 +76,15 @@ def find_tf_file(basedir):
 @click.option('--no-clean', 'no_clean', is_flag=True,
               help="Don't clean the environment when the tests finish. "
               "This is useful for debug and troubleshooting.")
-def cli(test, path, tfvar, snapshots, no_clean):
+@click.option('--loglevel', 'loglevel', type=click.Choice([
```

Comment:
is it not possible ?
```suggestion
@click.option('--loglevel', 'loglevel', type=click.Choice([logging.loglevel]),
```

Reply:
yes, if logging would have a attribute `loglevel` which is the list of all possible loglevels.

Reply:
ok